### PR TITLE
assignment 2 once again again

### PR DESCRIPTION
**Pull request by Ayannamdeo**

### CODE REVIEW

File: Assignment2/Q!.js
Comment (lovekesh-successive) [inline Assignment2/Q!.js:1]: Program will give the output 50 times, Fix this.

File: Assignment2/Q3.js
Comment (lovekesh-successive) [inline Assignment2/Q3.js:1]: make a generic function, if i want to subtract 4,1 how can i do it, fix this

**Review by lovekesh-successive — APPROVED** [review]

File: Assignment2/Q!.js
Comment (Ayannamdeo) [reply Assignment2/Q!.js:1]: fixed

File: Assignment2/Q3.js
Comment (Ayannamdeo) [reply Assignment2/Q3.js:1]: fixed

**Pull request merged**